docs/en/troubleshooting/Store operations/i-cant-index-a-product-in-the-catalog.md

```diff
@@ -13,10 +13,9 @@ slugEN: i-cant-index-a-product-in-the-catalog
 locale: en
 legacySlug: i-cant-index-a-product-in-the-catalog
 subcategoryId: 2Q0IQjRcOqSgJTh6wRHVMB
+tags: : Catalog, Index, Product, SKU, Trade policy
 ---
 
-**Tags:**: Catalog, Index, Product, SKU, Trade policy
```

Comment:
There is an extra `:` in line 16 only in this file, probably introduced by a double `:` coming from the original tags text (previously on line 18)

Reply:
Good catch! I'll fix it in Contentful as well.